ext/dom/dom_fe.h

```diff
@@ -67,163 +46,4 @@ typedef enum {
 	VALIDATION_ERR                 = 16
 } dom_exception_code;
```

Comment:
We might want to either move this somewhere else, or rename the file...

Reply:
Renamed to domexception.h